src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Ruby telemetry in context'
+metaDescription: For apps monitored by the Ruby agent, learn how to link log data with related data across the rest of the New Relic platform.
+---
+
+Telemetry in context for the Ruby agent connects your logs with all of your telemetry data for your apps, hosts, and other entities. Bringing all of this data together in a single tool helps you quickly:
+
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+![Telemetry in context in APM UI](./images/ruby-errors-log-details.png "Telemetry in context in APM UI")
+
+<figcaption>
+  **Telemetry in context:** From the left nav in the APM UI, you can select an error and immediately see related logs, log details, host name, important attributes, and more.
+</figcaption>
+
+## Enable and configure [#enable]
+
+You have three options to configure telemetry in context for your application.
+
+<CollapserGroup>
+
+  <Collapser
+    id="1-agent"
+    title="Option 1 (simplest): Let the agent decorate and forward your logs."
+  >
+
+  This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs and other telemetry in context without a lot of overhead.
+  
+  Using this option, your logs will include `span.id`, `trace.id`, `hostname`, `entity.guid`, and `entity.name`. These attributes will automatically link your logs with spans, traces, and errors for your app, its associated hosts, and other entities.
+
+  All you need to do is install an agent version with log forwarding capabilities ([Ruby agent 8.6.0 or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes/)), and update your configuration to enable forwarding.
```

Comment:
@tannalynn  - this is where you update the release notes link for 8.6.0

Reply:
Now that the release notes PR has been created. The link can be updated to 
https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-850/